app/views/admin/invoices/show.html.erb

```diff
@@ -1,2 +1,33 @@
-I made this show page for story 22 but we do not fill this page 
-out until 33
+<html>
+<head>
+  <title>Admin Invoice - <%= @invoice.id %></title>
+</head>
+<body>
+  <h1>Admin Invoice - <%= @invoice.id %></h1>
+  <p>Status: <%= @invoice.status %></p>
+  <p>Created at: <%= @invoice.created_at.strftime('%A, %B %d, %Y') %></p>
+  <p>Customer: <%= @invoice.customer.first_name %> <%= @invoice.customer.last_name %></p>
+
+  <h2>Items on Invoice:</h2>
+  <table class="invoice-table">
+    <thead>
+      <tr>
+        <th>Item Name</th>
+        <th>Quantity</th>
+        <th>Price</th>
+        <th>Invoice Item Status</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @invoice_items.each do |invoice_item| %>
+        <tr>
+          <td><%= invoice_item.item.name %></td>
+          <td><%= invoice_item.quantity %></td>
+          <td><%= number_to_currency(invoice_item.unit_price) %></td>
```

Comment:
Ooooooh, this is a clever method and I will definitely use it for some other things I'm making!

However, it looks like you called it before converting the unit_price from cents to dollars.

Reply:
done